listenbrainz/webserver/views/index.py

```diff
@@ -120,6 +124,35 @@ def current_status():
     )
 
 
+@index_bp.route('/delete-user/<musicbrainz_id>')
+def mb_user_deleter(musicbrainz_id):
```

Comment:
We decided in last week's meeting that storing the editor IDs in LB et al. is the way to go, so MB will actually send the editor row ID here and not the name. (Not sure if you've already started updating it to expect that, or if `musicbrainz_id` is actually the user name here.)